tests/test_local_executor.py

```diff
@@ -422,11 +422,7 @@ def test_finalize_failed_137(docker_cleanup, job_definition, tmp_work_dir, volum
     # impersonate an admin
     docker.kill(local.container_name(job_definition))
 
-    wait_for_state(api, job_definition, ExecutorState.ERROR)
-    assert (
-        api.get_status(job_definition).message
-        == "Job either ran out of memory or was killed by an admin"
-    )
+    wait_for_state(api, job_definition, ExecutorState.EXECUTED)
```

Comment:
yep, this path is not on this diagram https://github.com/opensafely-core/job-runner/blob/main/docs/local_executor_run.md but the `the job finishes with an error` has `ExecutorState.EXECUTED`

Reply:
I don't understand this comment I'm afraid? Is the diagram wrong now?

Reply:
nope all fine, this is just a by the way

Reply:
the diagram says this state should have been EXECUTED